tensorrt_llm/_torch/pyexecutor/kv_cache_manager_v2.py

```
@@ -1575,6 +1733,137 @@ def _log_kv_cache_pool_lifecycle_mapping(self) -> None:
+            without = _introspection.compute_slots_for_batch(self.impl, batch, tpb, None)
+            with_scratch = _introspection.compute_slots_for_batch(
+                self.impl, batch, tpb, scratch_config
+            )
+            capacities = [kv.capacity for kv in batch.kv_caches]
+            histories = [kv.history_length for kv in batch.kv_caches]
+            shape = (
+                f"{len(batch.kv_caches)} req(s), capacity={_summarize_ints(capacities)}, "
+                f"history={_summarize_ints(histories)}"
+            )
+            logger.info(f"  constraint[{idx}] ({shape}):")
+            for pg_idx, (no_s, yes_s) in enumerate(zip(without, with_scratch)):
```

Ruff (0.16.1)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)